curated_transformers/repository/hf_hub.py

```diff
@@ -0,0 +1,93 @@
+import warnings
+from typing import Optional
+
+import huggingface_hub
+from huggingface_hub.utils import (
+    EntryNotFoundError,
+    RepositoryNotFoundError,
+    RevisionNotFoundError,
+)
+from requests import HTTPError, ReadTimeout  # type: ignore
+
+from ..repository.file import LocalFile, RepositoryFile
+from .repository import Repository
+
+
+class HfHubRepository(Repository):
+    """
+    Hugging Face hub repository.
+    """
+
+    def __init__(self, name: str, *, revision: str = "main"):
+        """
+        :param name:
+            Name of the repository on Hugging Face Hub.
+        :param revision:
+            Source repository revision. Can either be a branch name
+            or a SHA hash of a commit.
+        """
+        self.name = name
+        self.revision = revision
+        self._model_config = None
+        self._tokenizer_config = None
```

Comment:
Neither of these appear to be used anywhere?

Reply:
Leftovers from less abstracted days. Fixed.